gamestonk_terminal.py

```diff
@@ -161,6 +191,7 @@ def print_help(s_ticker, s_start, s_interval, b_is_market_open):
     print("   clear       clear a specific stock ticker from analysis")
     print("   load        load a specific stock ticker for analysis")
     print("   view        view and load a specific stock ticker for technical analysis")
+    print("   export      export the currently loaded dataframe to a file or stdout")
```

Comment:
Can you add above: 
`if s_ticker:`

This is so that this command only appears once we have loaded a ticker, which I think is the right thing to do in this case